src/Sound/Tidal/UI.hs

```diff
@@ -809,7 +810,8 @@ euclidInv :: Pattern Int -> Pattern Int -> Pattern a -> Pattern a
 euclidInv = tParam2 _euclidInv
 
 _euclidInv :: Int -> Int -> Pattern a -> Pattern a
-_euclidInv n k a = fastcat $ fmap (bool a silence) $ bjorklund (n,k)
+_euclidInv n k a | n >= 0 = fastcat $ fmap (bool a silence) $ bjorklund (n,k)
```

Comment:
The implementation of this couldn't be:
```haskell
_euclidInv n k a = _euclid (-n) k a
```
?
it would avoid some duplication

Reply:
i think then we would have to replace the otherwise case in `_euclid` by the current definition of `_euclidInv`, no?

I agree that that might be a bit cleaner overall

Reply:
@polymorphicengine yes, probably at this point `euclidInv` could be also deprecated since it won't bring anything more than what `euclid` already bring